agent/consul/streaming_endpoint.go

```diff
@@ -128,11 +128,8 @@ func sendEvent(event stream.Event, aclFilter *aclFilter, eval *bexpr.Evaluator,
 	// hash it to save space. This is only used to determine if an object has been
 	// removed and needs to be deleted from the client's cache so we make the tradeoff
 	// of using less memory here at the extremely small risk of a hash collision.
-	rawId := event.ID()
-	hash, err := blake2b.New256(nil)
-	if err != nil {
-		return err
-	}
+	rawId := event.ContentID()
```

Comment:
I suggest for the comment above that we make it a little more precise. Something like:

```
	// Get the unique identifier for the object the event pertains to, and
	// hash it to save space. This is only used to determine if an object has been
	// removed and needs to be deleted from the client's cache. In other words, if we hit a
        // hash collision, it only results in a redundant message being sent without affecting 
        // correctness. 32 bits means there is only a 1% chance of collision with ~9000 items in this map.
        // For current uses that seems reasonable but when we add KV streaming we may want
        // to revisit this.
```